sessions/react-global-state/lights-solution/components/Lights/index.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import Light from "../Light";
+
+const StyledLights = styled.ul`
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 32px;
+  padding: 0;
+  margin: 0;
+  justify-content: center;
+`;
+
+export default function Lights({ lights, toggleLight }) {
+  return (
+    <StyledLights>
+      {lights.map(({ id, name, isOn }) => (
+        <li key={id}>
+          <Light name={name} isOn={isOn} onToggle={() => toggleLight(id)} />
```

Comment:
I don't understand why this solution works; I tried to pass `toggleLight` as `onToggle` to the `Light` component and call it there with the `id` argument, but this did not work.